include/zephyr/net/wifi_mgmt.h

```diff
@@ -729,6 +743,16 @@ enum wifi_mgmt_op {
 	WIFI_MGMT_SET = 1,
 };
 
+/** Wi-Fi 11k parameters */
+struct wifi_11k_params {
+	/** 11k command operation */
+	enum wifi_mgmt_op oper;
+	/** 11k enable/disable */
+	int enable_11k;
```

Comment:
For boolean type flags, please use `bool` type.

Reply:
Updated to` bool` type.